pycodestyle.py

```diff
@@ -270,6 +272,9 @@ def blank_lines(logical_line, blank_lines, indent_level, line_number,
                 yield 0, "E301 expected 1 blank line, found 0"
         elif blank_before != 2:
             yield 0, "E302 expected 2 blank lines, found %d" % blank_before
+    elif (logical_line and not indent_level and blank_before != 2 and
+          previous_logical_toplevel.startswith(('def', 'class'))):
+        yield 0, "E305 expected 2 blank lines before, found %d" % blank_before
```

Comment:
What about changing the message to be:

`E305 expected 2 blank lines after class or function definition, found %d`